recipes/HydroErr/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Please _remove_ the license file from this feedstock and add it to the package's source distribution instead (by creating a `MANIFEST.in` file in the https://github.com/BYU-Hydroinformatics/HydroErr repo)

Reply:
We can merge with the license here as long as PR is submitted upstream.